src/toolbar/sticky/stickytoolbarview.js

```diff
@@ -20,21 +23,79 @@ export default class StickyToolbarView extends ToolbarView {
 	/**
 	 * @inheritDoc
 	 */
-	constructor() {
-		super();
+	constructor( locale ) {
+		super( locale );
 
 		const bind = this.bind;
 
 		this.model.set( 'isSticky', false );
+		this.model.set( '_isStickyToLimiterBottom', false );
+		this.model.set( 'left', null );
+		this.model.set( 'marginLeft', null );
+		this.model.set( 'limiterElement', null );
 
 		Template.extend( this.template, {
 			attributes: {
-				// Toggle class of the toolbar when "sticky" state changes in the model.
-				class: bind.if( 'isSticky', 'ck-toolbar_sticky' )
+				class: [
+					// Toggle class of the toolbar when "sticky" state changes in the model.
+					bind.if( 'isSticky', 'ck-toolbar_sticky' ),
+					bind.if( '_isStickyToLimiterBottom', 'ck-toolbar_sticky_bottom-limit' ),
```

Comment:
I guess – "I'm on the limit – position me absolutely". So when any other styling doesn't make much sense, then perhaps adding `pos:abs` inline would make sense too? OTOH, perhaps someone will want to style such a toolbar a bit differently.